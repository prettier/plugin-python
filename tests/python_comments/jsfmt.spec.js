run_spec(__dirname, ["python"], { pythonVersion: "2" });
run_spec(__dirname, ["python"], { pythonVersion: "3" });
