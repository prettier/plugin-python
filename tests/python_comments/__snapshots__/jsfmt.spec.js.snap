// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`comments.py 1`] = `
if a:  # a
    # b
    c

    # c

    d

    # e

    call(
        # a
    )
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
if a: # a
    # b
    c

    # c

    d

    # e

    call()
    # a

`;

exports[`comments.py 2`] = `
if a:  # a
    # b
    c

    # c

    d

    # e

    call(
        # a
    )
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
if a: # a
    # b
    c

    # c

    d

    # e

    call()
    # a

`;
